packages/react-lowcode/src/codegen/generation/generators/list/mui-datatable-generator.ts

```diff
@@ -254,6 +255,63 @@ export default class MuiDataTableGenerator implements TableGenerator
       }else return undefined
     }
 
+    generateTablePage(template: string): PageComponent | undefined {
+      let result: PageComponent | undefined;
+
+      if (this._entity) {
+        let ast = createAst(template);
+
+        if (ast) {
+          const templateParenthesizedExpression = find<ParenthesizedExpression>(ast, (node: Node) => {
+            return ts.isParenthesizedExpression(node);
```

Comment:
This seems to be weak and not generic enough. I believe we should find `ListPlaceholder `.

Reply:
It was refactored in meantime.